src/SfxWeb/cypress/integration/cluster.spec.js

```diff
@@ -188,19 +188,24 @@ context('Cluster page', () => {
 
       cy.get('[data-cy=eventtabs]').within(() => {
         cy.contains(CLUSTER_TAB_NAME)
-        cy.contains(REPAIR_TASK_TAB_NAME)
       })
     })
 
     it("repair manager disabled", () => {
+      addRoute('events', 'empty-list.json', apiUrl(`/EventsStore/Cluster/Events?*`))
       addRoute('repair-manager-manifest', 'manifestRepairManagerDisabled.json', manifest_route)
 
       cy.visit('/#/events')
-
-      cy.wait('@getrepair-manager-manifest')
+      cy.wait(['@getevents','@getrepair-manager-manifest'])
 
       cy.get('[data-cy=eventtabs]').within(() => {
         cy.contains(CLUSTER_TAB_NAME)
+      })
+      checkTableErrorMessage(EMPTY_LIST_TEXT);
+
+      cy.get('[sectionName=select-event-types]').click()
```

Comment:
given these strings show up multiple times make them a variable at the top of this file.